closed/autoconf/custom-hook.m4

```diff
@@ -405,6 +405,13 @@ AC_DEFUN_ONCE([CUSTOM_LATE_HOOK],
 
   # explicitly disable classlist generation
   ENABLE_GENERATE_CLASSLIST=false
+
+  if test "x$OPENJDK_BUILD_OS" = xwindows ; then
+    OPENJ9_TOOL_DIR="$OUTPUTDIR/tools"
+    AC_SUBST([OPENJ9_TOOL_DIR])
+    OPENJ9_GENERATE_TOOL_WRAPPERS
```

Comment:
I don't see anything making this specific to cmake builds.

Reply:
I can change it if you want. I figured it really didn't matter if we generate a few text files that potentially go unused